src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/styles.ts

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+import { StyledTagProps } from './types';
+
+export const StyledIcon = styled(Icon)`
+  height: 1.2rem;
```

Comment:
Reduce the icon size and overall tag height and width, it looks too big to me.

Reply:
About half of the original?

Reply:
Yeah, try it, if it doesn't look too small.

Reply:
![Screenshot from 2020-04-18 00-49-52](https://user-images.githubusercontent.com/43489708/79606056-a0788680-810e-11ea-8fd7-d757263bbe51.png)

This is 1rem height of styled icon, will it do? I tried doing it even smaller, but it looked too small then.

Reply:
Yes, this size is fine.

Reply:
Done